internal/scheduled/event.go

```diff
@@ -0,0 +1,27 @@
+package scheduled
+
+import (
+	"time"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/dynamo"
+)
+
+// A Event specifies an action to take in the future.
+type Event struct {
+	PK dynamo.ScheduledDayKeyType
+	SK dynamo.ScheduledKeyType
+	// CreatedAt is when the Row was created
+	CreatedAt time.Time
+	// LockedUntil is set to show the Row is running, it cannot be retrieved until this time has passed
+	LockedUntil time.Time
+	// HandledAt is set when the Row has finished running
+	HandledAt time.Time
```

Comment:
Are these being used yet?

Reply:
ah nope, these are leftover from a previous way I was going to do things